lms/djangoapps/verified_track_content/tests/test_models.py

```diff
@@ -47,6 +47,20 @@ def test_unicode(self):
         config.save()
         self.assertEqual(unicode(config), "Course: {}, enabled: True".format(self.SAMPLE_COURSE))
 
+    def test_verified_cohort_name(self):
+        COHORT_NAME = 'verified cohort'
+        course_key = CourseKey.from_string(self.SAMPLE_COURSE)
+        config = VerifiedTrackCohortedCourse.objects.create(
+            course_key=course_key, enabled=True, verified_cohort_name=COHORT_NAME
+        )
+        config.save()
+        self.assertEqual(VerifiedTrackCohortedCourse.verified_cohort_name_for_course(course_key), COHORT_NAME)
```

Comment:
Can you also add a test for the default name (verified_cohort_name not specified)?